### ISSUE TITLE
Add configurable donation/charity output address in for jobmanagers

### ISSUE THREAD

**Issue by ericecook** [simplecrypto/powerpool#109]
Several coins (including EMC2) require some portion of the coinbase reward to be paid out to a specific charity address, and PowerPool doesn't currently support this functionality (AFAIK)

All that is needed is configuring the address, and optionally the key to look for in the `getblocktemplate` response. For the ones I've looked at the required value is provided via a `charityvalue`.

This should be relatively simple to implement, and allow mining quite a few more coins.

Afterthought: It might be possible to cleanly include darkcoin's masternode payout in this code block.

**Comment by icook** [comment]
Ideally this would be something that would fall to cckit. Job generation would be currency specific functionality for a coin.

In the interim however patching it in might be worth it. Unsure.

**Comment by ericecook** [comment]
I'm a bit torn. My thinking was that having it be a directly configurable value would be more coin agnostic - and therefore a bit more flexible for the operator. But eventually I think it'd be better handled by cckit

**Comment by icook** [comment]
Yeah, I'll put in a few more days on cckit and see how close it is to acting as a replacement and reeval. Certainly if we want to expand our coin offerings then this is an important issue.
